### ISSUE TITLE
Misinformation

### ISSUE THREAD

**Issue by Lawful-Lazy** [lmanul/whylinuxisbetter#18]
Lots of misinformation without any citations. Without any citations or proof of working, then there is bound to have misinformation. [Here's a video that shows some parts of misinformation on this site.](https://www.youtube.com/watch?v=y_lhqg_p21k)

_One of the biggest flaws of any device is the misinformation about that device..._

**EDIT:** Made Pull Request #19 - Please do add comments and suggestions there! Thanks!

**Comment by aaronfranke** [comment]
That video is not a good source of information at all. For example, for the libraries 'that aren't updated until you reboot', you don't need to restart the system, you just need to kill all the processes using it.

For the fragmentation issue, it is true that EXT4/other Linux filesystems DO fragment, but the point that he don't get through is that the ext filesystem is much less prone to fragmentation than NTFS is (which is a different base design).

Now, for the virus thing, yes Linux can get viruses, but the system is inherently more secure than windows, as the access management is much more restricting, partly due to the fact that Linux was originally used as a server OS and was built to handle multiple users and files with ownership (and also because of other stuff, one of them being the open source system, which allow anyone to correct any security problem and audit the code in the same goal). Also, in his particular case, he had Windows viruses (.exe, .dll) in a temp folder, it seems that he may have gotten crapware that was bundled by SourceForge in projects at around the time his video was done (and those aren't Linux crapware). His software looks in the temp folder and tries to load things in there by that program's design.

For the drive stuff, it comes from the fact that Arch uses a UUID system based on the whole computer for identifying the disk drives, they would actually just need to be regenarated for the system to be usable again.

Overall, it does look like he knows some stuff, but not enough to get why the stuff is happening.

**Comment by MathManiac0** [comment]
While I don't go full-on what he is saying, he still has some good points (including why there are *three* defragmenters in Linux).

That's why, in my Pull Request ( #19 ), I did add some of the facts, but not everything, such as:
- Not updating until you restart. He even showed you didn't need to restart it!
- Drive stuff. I ***barely*** understand a single thing about the drivers, so - to keep it safe - I didn't add it in my pull request.
- Fragmentation. While I *did* add some stuff, I also made sure to say that - for the average user - **it doesn't need defragmenting.**
- Viruses. Of course I know that Linux is much more secure. I even added that since less than 2% of people use Linux (In a study on Aug. 2015), rarely anyone should be bothered to create Linux - especially with multiple prying eyes on the Linux code looking for security fixes, etc.

In short, I want to make sure that this website isn't advertising Linux as an "Everything is perfect" OS, but it can be still far superior in many ways.

Oh - and for convenience of myself and anybody reading this - Adding the rest of comments on my Pull Request would be really great! Thanks!

**Comment by aaronfranke** [comment]
Yes I agree with that. However it's worth noting that <2% of people use *desktop* Linux. Most people use other forms of Linux on a daily basis (phones, servers, routers, embedded devices).

**Comment by MathManiac0** [comment]
In context to this website, what kind of Linux do you think we're talking
about?

**Comment by lmanul** [comment]
This sounds more like a heated discussion starter than a set of constructive criticisms ;-P If you have reasons to think that some of the information on the site is objectively wrong, please feel free to send pull requests with some comments explaining why and giving some sources, I'd be happy to fix anything that's provably wrong!
